ci/jenkins/test-vmc.sh

```diff
@@ -700,6 +718,11 @@ if [[ "$TESTCASE" == "integration" ]]; then
     exit 0
 fi
 
+if [[ "$TESTCASE" == "multicluster-integration" ]]; then
```

Comment:
Make some changes according to the change above.

Reply:
done, @lzhecheng Could you help to review again? thanks.